lxd/resource_lxd_container.go

```diff
@@ -85,6 +90,44 @@ func resourceLxdContainer() *schema.Resource {
 				ForceNew: false,
 			},
 
+			"file": &schema.Schema{
+				Type:     schema.TypeList,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"content": &schema.Schema{
+							Type:     schema.TypeString,
+							Required: true,
+						},
+
+						"target_file": &schema.Schema{
+							Type:     schema.TypeString,
+							Required: true,
+						},
+
+						"uid": &schema.Schema{
+							Type:     schema.TypeInt,
+							Optional: true,
+						},
+
+						"gid": &schema.Schema{
+							Type:     schema.TypeInt,
+							Optional: true,
+						},
+
+						"mode": &schema.Schema{
+							Type:     schema.TypeString,
```

Comment:
I considered int, but went with string for two reasons:

1. It matches LXC, including parsing to int and such.
2. It enables modes like "0644", even though a zero is added later if missing.

I'm happy to change it to int, though.